raiden/network/transport/udp/udp_transport.py

```diff
@@ -543,7 +545,20 @@ def receive_delivered(self, delivered: Delivered):
         protocol, but it's required by this transport to provide the required
         properties.
         """
-        processed = ReceiveDelivered(delivered.delivered_message_identifier)
+        for queue_identifier, events in self._queueids_to_queues.items():
```

Comment:
Perhaps we should change `self._queueids_to_queues` to `channelids_to_queues = self.recipients_to_channelqueues[recipient]`? (This can be done in another PR though)

Reply:
If i understand you correctly, it's not just a rename but more like also getting rid of the "queue_identifier" to be replaced by the channel_identifier to get queued messages?

Reply:
This replacement will be done as part of #2537